GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -162,6 +162,32 @@ public virtual void InitializeDefaultParameterValues()
             return verb.ReturnCode;
         }
 
+        protected ReturnCode Execute<TVerb>(
+            GVFSEnlistment enlistment,
+            Action<TVerb> configureVerb = null)
+            where TVerb : GVFSVerb.ForExistingEnlistment, new()
+        {
+            TVerb verb = new TVerb();
+            verb.EnlistmentRootPathParameter = enlistment.EnlistmentRoot;

```

Comment:
Not sure if intentional, but there is a subtle change in behavior here.  

Previously `EnlistmentRootPathParameter` would be set to `fullEnlistmentRootPathParameter` and now it will be `normalizedEnlistmentRootPath`.

I just wanted to make sure this is OK (I suspect the potential impact would be on the paths that users see when these aren't the same).

Reply:
Yep I had noticed this change and my thought was that it's pretty minor. In the end, the user will see the normalized form of the same path they specified, which I don't think is harmful. Any concerns?

Reply:
I don't think it will be harmful either.  I think we can keep this change and see if we get any feedback on it (which I don't expect we will).